packages/spotlight/SpotlightContainerDecorator/SpotlightContainerDecorator.js

```diff
@@ -116,6 +116,14 @@ const SpotlightContainerDecorator = hoc(defaultConfig, (config, Wrapped) => {
 	const forwardMouseLeave = forward(leaveEvent);
 	const {navigableFilter, preserveId, ...containerConfig} = config;
 
+	const add = ({spotlightId}) => {
```

Comment:
`add` is not good for the function name here.
How about `initializeState`?

Reply:
@MikyungKim and I have no idea for naming. If 2nd-reviewer has the same opinion to rename it, please let @MikyungKim know.

Reply:
Maybe `stateFromProps`?

Reply:
Well, this is not from my head so I'd like to hear from the original author, @ryanjduffy .

Reply:
Might want to proactively migrate to a static `getDerivedStateFromProps` (or something similar) in prep for React 16.4. If we wanted to be safe, we could define it as a `const` as it is now and migrate it to a static member when we complete the 16.3/4 adoption.